src/modules/75-cf/pages/feature-flags/__tests__/FeatureFlagsPage.test.tsx

```diff
@@ -60,6 +60,9 @@ describe('FeatureFlagsPage', () => {
     mockImport('services/cf', {
       useGetAllFeatures: () => ({ data: mockFeatureFlags, refetch: jest.fn() })
     })
+    mockImport('services/cf', {
+      getFeatureMetricsPromise: () => Promise.resolve(undefined)
+    })
```

Comment:
Not a blocker, but probably should combine these
```suggestion
    mockImport('services/cf', {
      useGetAllFeatures: () => ({ data: mockFeatureFlags, refetch: jest.fn() }),
      getFeatureMetricsPromise: jest.fn().mockResolvedValue(undefined)
    })
```

Reply:
good call, will fix next